### PR TITLE
refactor : make home page responsive

### PR DESCRIPTION
**Pull request by viru2001**
## make home page responsive

responsive home page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -35,36 +35,36 @@ <h1 class="headline-lg m-3 logo">
...
-            <a href="#" class="no-style-link text-sm"> Installation </a>
+            <a href="https://github.com/viru2001/Cool-UI" target="_blank" class="no-style-link text-sm"> Github </a>
           </button>
         </div>
       </div>
 
-      <div class="promotional-image-wrapper">
-        <img class="w-75" src="./assets/landing-page-illustartion.svg" alt="" />
+      <div class="promotional-image-wrapper m-6">
+        <img id="promo-img" class="img-responsive" src="./assets/landing-page-illustartion.svg" alt="" />
```
Comment (surya-nemiwal) [inline index.html]: can give descriptive value to alt attribute like "Promo image"

Comment (viru2001) [reply index.html]: ok will add it

**Review by surya-nemiwal — APPROVED** [review]
Looks nice to be merged. Great Job.

**Review by poojakelkar — APPROVED** [review]
Looks good to merge for me

**Pull request merged**